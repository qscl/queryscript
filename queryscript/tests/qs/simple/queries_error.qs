-- Unsupported
select u2.* from users join users u2 on users.id = u2.id;

-- Should error
select * from users join users on true;
select count(*) from users, users u2 where users.id = u2.a;
select description, (select * from users where users.id = events.user_id) from events order by ts;
WITH u AS (SELECT * FROM users), u AS (SELECT * FROM users) SELECT * FROM u;
