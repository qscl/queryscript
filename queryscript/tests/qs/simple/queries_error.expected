{
    "compile_errors": [
        (
            Some(
                0,
            ),
            NoSuchEntry {
                path: [
                    "users",
                ],
                backtrace: None,
            },
        ),
        (
            Some(
                1,
            ),
            NoSuchEntry {
                path: [
                    "users",
                ],
                backtrace: None,
            },
        ),
        (
            Some(
                2,
            ),
            NoSuchEntry {
                path: [
                    "users",
                ],
                backtrace: None,
            },
        ),
        (
            Some(
                3,
            ),
            NoSuchEntry {
                path: [
                    "events",
                ],
                backtrace: None,
            },
        ),
        (
            Some(
                4,
            ),
            NoSuchEntry {
                path: [
                    "users",
                ],
                backtrace: None,
            },
        ),
    ],
    "decls": {},
    "queries": [],
}